Source/WebCore/html/ColorInputType.cpp

```diff
@@ -200,6 +200,9 @@ Color ColorInputType::valueAsColor() const
     ASSERT(element());
     auto color = parseColorValue(element()->value(), *element());
     ASSERT(!!color);
+    // FIXME: This is a speculative fix for rdar://144872437.
+    if (!color)
+        return Color::black;
```

Comment:
Why not returning the default `Color { }` ? It is invalid and could be dealt by later code in whatever way is locally appropriate, contrary to hardcoding `black` ?

Reply:
Black is the fallback color for the color control.